migration/utils/utils.go

```diff
@@ -0,0 +1,38 @@
+package migrationutils
+
+import (
+	"crypto/rand"
+	"os"
+	"path/filepath"
+)
+
+// RandomSlice returns a randomly filled byte array with length of given size
+func RandomSlice(size int) (out []byte) {
```

Comment:
we do have another function in utils right ?

Reply:
Yes we do, the main reason to duplicate is that we do not want to add any dependencies if possible with the code base, as we could extend this package and make it more configurable and its own repository for go-(leveldb-otherdb) migrations